controllers/monitoring_alert.go

```diff
@@ -528,3 +539,36 @@ func (r *ManagedMCGReconciler) reconcilePrometheusProxyNetworkPolicy() error {
 
 	return nil
 }
+
+func (r *ManagedMCGReconciler) configureRHOBSRemoteWrite() error {
+	if err := r.get(r.rhobsRemoteWriteConfigSecret); err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+	if r.rhobsRemoteWriteConfigSecret.UID == "" {
+		r.prometheus.Spec.RemoteWrite = nil
+		r.Log.Info("RHOBS remote write config secret not found, disabling remote write")
+
+		return nil
+	}
+	rhobsSecretData := r.rhobsRemoteWriteConfigSecret.Data
+	if _, found := rhobsSecretData[rhobsRemoteWriteConfigIDSecretKey]; !found {
+		return fmt.Errorf("rhobs secret does not contain a value for key %v", rhobsRemoteWriteConfigIDSecretKey)
+	}
+	if _, found := rhobsSecretData[rhobsRemoteWriteConfigSecretName]; !found {
+		return fmt.Errorf("rhobs secret does not contain a value for key %v", rhobsRemoteWriteConfigSecretName)
+	}
+	rhobsAudience, found := rhobsSecretData["rhobs-audience"]
+	if !found {
+		return fmt.Errorf("rhobs secret does not contain a value for key rhobs-audience")
+	}
+	remoteWriteSpec := &r.prometheus.Spec.RemoteWrite[0]
```

Comment:
@rexagod one doubt, remoteWriteSpec defined and values assigned to it but I couldn't find actual usage of that spec?

Reply:
Right.

Reply:
This does not seem to be used anywhere.

![image](https://user-images.githubusercontent.com/33557095/182120726-6f924807-e74b-4c9b-8042-95a92ec6658d.png)

Reply:
@jmolmo PTAL.

Reply:
The remote write config is defined in templates/prometheus.go (line 116). In monitoring_alert.go is updated with the configured values

Reply:
@jmolmo remoteWriteSpec pointing to spec memory, Thanks for clarifying this.